ruby/vampires.rb

```diff
@@ -0,0 +1,76 @@
+puts "How many employees will be processed today?"
+employee_total = gets.to_i
+
+while employee_total > 0 do
+  puts "What is your name?"
+  name = gets.chomp
+
+  puts "How old are you?"
+  age = gets.chomp.to_i
+
+  puts "What year were you born?"
+  year = gets.chomp.to_i
+  current_year = age + year
+  if current_year == 2016 || current_year == 2015

```

Comment:
This can be handled with the Ruby `Time` class see how you can use `Time.now.year` to create a dynamic solution to checking the year.